miner/bid_simulator.go

```diff
@@ -593,6 +610,34 @@ func (b *bidSimulator) simBid(interruptCh chan int32, bidRuntime *BidRuntime) {
 		return
 	}
 
+	// fill transactions from mempool
+	if b.config.AppendToBid {
```

Comment:
AppendToBid seems not a good name

Reply:
how about enlargeBid?

Reply:
I dont have good idea either... What about `LocalTxMerge`, `GreedyMergeTx`?

Reply:
ok, GreedyMergeTx